src/core/ext/filters/client_channel/resolver/dns/c_ares/grpc_ares_wrapper.cc

```diff
@@ -522,7 +522,7 @@ grpc_error_handle grpc_ares_ev_driver_create_locked(
     int query_timeout_ms,
     std::shared_ptr<grpc_core::WorkSerializer> work_serializer,
     grpc_ares_request* request) {
-  *ev_driver = new grpc_ares_ev_driver();
+  grpc_ares_ev_driver* driver = *ev_driver = new grpc_ares_ev_driver();
```

Comment:
Suggestion: it might be worth pushing just this fixit to master. Good cleanup, not totally related to EE.